local-rest-scorer/src/main/java/ai/h2o/mojos/deploy/local/rest/config/JsonConfiguration.java

```diff
@@ -1,17 +1,20 @@
 package ai.h2o.mojos.deploy.local.rest.config;
 
 import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.core.JsonParser;
 import com.fasterxml.jackson.databind.ObjectMapper;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
+import org.springframework.http.converter.json.Jackson2ObjectMapperBuilder;
 
 @Configuration
 class JsonConfiguration {
 
   @Bean
   public ObjectMapper objectMapper() {
-    ObjectMapper mapper = new ObjectMapper();
-    mapper.setSerializationInclusion(JsonInclude.Include.NON_NULL);
-    return mapper;
+    return new Jackson2ObjectMapperBuilder()
```

Comment:
qq - I believe this is the same for aws lambda as well... maybe needs change there also??

Reply:
Good point, but lambda is using gson (not the Spring objectmappers/jackson). But will need test as well.